### PR TITLE
Generate list of all extbans.

### PR DESCRIPTION
**Pull request by ProgVal**
This time I included descriptions in the table, because they fit. (It's also more important than for modes, as extbans aren't named)

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ title: Introduction
     - [Upgrading from v2](/3/breaking-changes)
 - [Commands](/3/commands)
 - [Channel Modes](/3/channel-modes)
+- [Extended Bans](/3/extended-bans)
```
Comment (SadieCat) [inline docs/index.md:31]:
```suggestion
  - [Extended Bans](/3/extended-bans)
```

File: docs/3/extended-bans.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: v3 Extended Bans
+---
+
+## Extended Bans
+
+Some [Channel modes](/3/channel-modes/) typically take a nick or `<nick>!<user>@<host>` as parameter. Extended Bans are extensions to the possible parameters, to either allow either more advanced selection of who these modes affect ("Matching" type), or change the meaning of the mode ("Acting" type).
```
Comment (SadieCat) [inline docs/3/extended-bans.md:7]: This feels like it could be worded better but I'm not sure how hm.

File: docs/index.md (hunk 1 above)
Comment (ProgVal) [reply docs/index.md:31]: I don't see any change in the output when doing this

File: docs/3/extended-bans.md (hunk 2 above)
Comment (ProgVal) [reply docs/3/extended-bans.md:7]: sam

File: docs/index.md (hunk 1 above)
Comment (ProgVal) [reply docs/index.md]: got it, it needed more indent.

Comment (SadieCat) [reply docs/index.md]: Ah sorry it should be four spaces not two.

File: docs/3/extended-bans.md (hunk 2 above)
Comment (DanielOaks) [reply docs/3/extended-bans.md:7]:
```markdown
Some [channel modes](/3/channel-modes/) (such as 'ban' or 'invite exception') accept nicknames or hostmasks as parameters. Using the extended ban parameters listed here lets you either match clients in more complex ways ('matching' extbans) or changes the meaning of the mode ('acting' extbans).

This lets you, for example, block CTCPs from matched users, match a specific user by their account name, and get more fine-grained control over how clients are moderated in a channel.
```

idk maybe?

**Pull request merged**